app/controllers/admin/legislation/processes_controller.rb

```diff
@@ -24,7 +24,7 @@ def create
 
   def update
     if @process.update(process_params)
-      set_tag_list
+      set_tag_list if process_params[:custom_list]
```

Comment:
@andrewcameronsims Awesome! :tada: Nice debugging there to find the problem! :+1: And great test! :clap:

I've just realized we don't even need the `set_tag_list` method at all; the `acts_as_taggable_on :customs` line in `Legislation::Process` generates the `custom_list=` method, and the attribute is correctly updated when we call `@process.update(process_params)`.

So would you mind removing the `set_tag_list` method completely? :pray:

By the way, are you familiar with `git rebase`? If so, could you squash all commits into one? For such a small pull request, having four commits seems too much :thinking:. If not, it's alright :wink:.

Reply:
No worries. Is this OK? Thanks for the feedback.